defaults.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should these include a commented-out example?

Reply:
JSON doesn't support comments 🤬